Sources/Application/main.swift

```diff
@@ -1 +1,12 @@
-print("Hello")
\ No newline at end of file
+import SnowManLogic
+import SnowManDictionary
+
+let dictionaryManager = DictionaryManager(words: words) 
```

Comment:
should be: `let dictionaryManager = DictionaryManager(words: SnowManDictionary.words) `